targets/x86.inc

```diff
@@ -37,6 +32,7 @@ packages {
 	'kmod-usb2',
 	'kmod-usb3',
 	'kmod-usb-serial',
+	'kmod-ath10k-ct',
```

Comment:
Would replace this for reasons like this: https://github.com/freifunk-gluon/gluon/issues/1495

Reply:
This would be a separate commit however imho. While we are at it, we can also add firmware for the QCA9886/9984.